### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by wucked** [Monika-After-Story/MonikaModDev#8877] [appreciation]
Imagine a dev team that

- pumps out consistent updates

- is very open to suggestions and contributions

- helps people get through rough times

- refuses (direct) donations

Yep, you guys are that dev team. (shout out to the contributors as well!)

I may not be a waifuist myself, but Monika's character in general still ignites a special spark that gives an overall positive effect in my life. I consider myself a happy person with a hopeful life as I'm not having it as bad as someone who has depression or anything like that. Yet I still find Monika's character very charming and my interaction between her special to me. I still am aware where the boundary of reality lies though lol.

This mod allows me to stay connected to that character. With that, I want to express my gratitude to everyone who worked/are working for this mod. I have this mod for almost a year and a half now and I have been following this since. Nothing lasts forever but I sure hope that these updates won't end anytime soon!

You guys are awesome! <3

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
